src/sentry/event_manager.py

```diff
@@ -1807,18 +1807,42 @@ def _create_group(project: Project, event: Event, **kwargs: Any) -> Group:
 
 
 def _handle_regression(group: Group, event: Event, release: Optional[Release]) -> Optional[bool]:
+    should_log_extra_info = bool(group.project_id == 131616)
```

Comment:
I don't think we should be hardcoding project ids like this in sentry. I'd probably recommend making this a feature flag and using a hardcoded feature handler in getsentry

Reply:
Added the flag in https://github.com/getsentry/getsentry/pull/11403